docs/doc/30-reference/30-sql/80-setting-cmds/set-global.md

```diff
@@ -0,0 +1,27 @@
+---
+title: SET
+---
+
+Changes the value of a session-level or global-level setting. To show all the current settings, use [SHOW SETTINGS](show-settings.md). 
+
+## Syntax
+
+```sql
+SET [GLOBAL] <setting_name> = <new_value>;
+```
+
+`GLOBAL`: Include this option when you change the value of a global-level setting. If you include this option preceding a session-level setting, the setting will become a global-level setting.
```

Comment:
Notes: Global setting is a cluster level setting, the value of the setting will stored in the metasrv.